src/styles/blocks/main.scss

```diff
@@ -0,0 +1,102 @@
+.page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --star-size: 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 0 0;
+  }
+
+  &__title {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-accent;
+    width: 166px;
+    margin: 40px 0 0;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    width: 166px;
```

Comment:
No need to hardcode width 166px everywhere, it will be better if you set internal indents to the card, and all these elements will stretch to the entire available width
```suggestion
```